spynnaker/pyNN/spynnaker.cfg

```diff
@@ -29,9 +29,9 @@ one_to_one_connection_dtcm_max_bytes = 0
 # BasicRoutingInfoAllocator, BasicDijkstraRouting,
 # MallocBasedRoutingInfoAllocator, GraphEdgeFilter, EdgeToNKeysMapper
 
-application_to_machine_graph_algorithms = PartitionAndPlacePartitioner
+application_to_machine_graph_algorithms = BasicNAtomsPartitioner
```

Comment:
Clearly this makes quite a big change to master!  Maybe having this as a commented out version would make more sense...